### PR TITLE
feat: New Margin for Footer & added 2022 for COSCUP

### PR DESCRIPTION
**Pull request by steveyiyo**

### CODE REVIEW

**Comment by Haraguroicha** [comment]
why you edited many unrelated lines for this feature and margin fix

**Comment by steveyiyo** [comment]
That is just auto-format by VSCode Plugin. I've disabled and forced push commit again.

Thanks!

File: public/css/main.css [hunk 1]
```diff
@@ -82,4 +86,4 @@ html[data-useragent*="AppleWebKit/"][data-useragent*="Safari/"]:not([data-userag
 
 #appsrc.one {
     width: 100%;
-}
+}
```
Comment (Haraguroicha) [inline public/css/main.css]: here still edited

Comment (steveyiyo) [reply public/css/main.css]: Resolved, Thanks

File: public/index.html [hunk 2]
```diff
@@ -132,10 +132,12 @@ <h4 class="ui horizontal container divider header">
...
-                <i class="info circle icon"></i>
-                &nbsp;OPass 2016 - {{nowYear}} |
-                <a href="privacy-policy.html">Privacy Policy</a> |
-                <a href="mailto:dev@opass.app">Contact Us</a>
+                <div class="footer">
+                    <i class="info circle icon"></i>
+                    &nbsp;OPass 2016 - {{nowYear}} |
+                    <a href="privacy-policy.html">Privacy Policy</a> |
+                    <a href="mailto:dev@opass.app">Contact Us</a>
+                </div>
```
Comment (Haraguroicha) [inline public/index.html]: so here still edited for what?

Comment (steveyiyo) [reply public/index.html]: Added `<div class="footer">` for footer with margin.

Comment (Haraguroicha) [reply public/index.html]: do you know you can just add that style into h4 element by a new class name ?

**Pull request closed without merging**